src/Interpreter.test.ts

```diff
@@ -3468,3 +3469,26 @@ test('choctaw 2020 general', async () => {
     ]
   `)
 })
+
+test('normalizes intentionally empty pages correctly', async () => {
+  const fixtures = choctaw2020Special
+  const { election } = fixtures
+  const interpreter = new Interpreter(election)
+
+  await interpreter.addTemplate(await fixtures.blankPage1.imageData())
```

Comment:
is this a blank first page? Should it be a normal first page to be a true test?

Reply:
The terminology is a little inconsistent. This "blank" means "unmarked", as in a template page.

Reply:
ah got it, so the other page is actually empty / no-contests? cool.